wof/WaterML_1_1.py

```diff
@@ -18,14 +18,14 @@
 #   WOFpy
 #
 
-from __future__ import (absolute_import, division, print_function)
+from __future__ import (absolute_import, division, print_function, unicode_literals)
 
 import sys
-import re as re_
+import re
 import base64
-import datetime as datetime_
-import warnings as warnings_
-from lxml import etree as etree_
```

Comment:
Sorry for the noise. I just never got why the original imports were renamed to something strange.
Maybe someone trying to make the code more obscure to his job :laughing: